src/components/Filters/components/ConnectedFilterControl/ConnectedFilterControl.tsx

```diff
@@ -69,13 +70,23 @@ export class ConnectedFilterControl extends React.Component<
       rightPopoverableActions,
       rightAction,
       auxiliary,
+      hideMoreFiltersButton,
     } = this.props;
 
     const className = classNames(
       styles.ConnectedFilterControl,
       rightPopoverableActions && styles.right,
     );
 
+    // If there's only one action, and it fits we don't show the "More filters" button
+    const moreFilterButton =
+      rightPopoverableActions &&
+      rightPopoverableActions.length === 1 &&
```

Comment:
Hmm.. It shouldn't be based on the number 1 ;)

If all actions are "inlined", the "more filters" button becomes useless.
So if `rightPopoverableActions.length === this.getActionsToRender(rightPopoverableActions).length`, then we shouldn't render the button I believe.

Reply:
Currently, when you click on `More filters` while being on mobile you get the following:
![image](https://user-images.githubusercontent.com/3137417/76996260-96087880-6927-11ea-8ae5-4363951ef2db.png)

Do you suggest keeping the Desktop behaviour if `rightPopoverableActions.length === this.getActionsToRender(rightPopoverableActions).length` is true? As so:

![image](https://user-images.githubusercontent.com/3137417/76996299-a91b4880-6927-11ea-89b0-b304bf056c74.png)